fastlane/lanes/release.rb

```diff
@@ -24,28 +24,25 @@
 
     release_branch_name = compute_release_branch_name(options:, version: release_version_next)
 
-    skip_user_confirmation = options[:skip_confirm]
```

Comment:
As discussed here, https://github.com/wordpress-mobile/WordPress-iOS/pull/22191#discussion_r1424458459, reverted the change that printed this information only in user interactive (`skip_confirm:false`) mode.